tests/manual-tests/framework/preferences/assortedContent/index-jQueryUI.html

```diff
@@ -0,0 +1,299 @@
+<!DOCTYPE html>
```

Comment:
Instead or as well as making using this manual test, maybe creating a UIO or PrefsFramework example would be good. https://github.com/fluid-project/infusion/tree/master/examples